lib/puppet/provider/scheduled_task/taskscheduler_api2.rb

```diff
@@ -147,8 +146,7 @@ def trigger=(value)
     end
 
     needed_triggers.each do |trigger_hash|
-      v1trigger = PuppetX::PuppetLabs::ScheduledTask::Trigger::V1.from_manifest_hash(trigger_hash)
-      task.append_trigger(v1trigger)
+      task.append_trigger(trigger_hash)
```

Comment:
after the rename above, should `trigger_hash` be changed to `manifest_hash` or vice versa or left alone?

Reply:
Yeah, I think that makes sense.

Reply:
Will save this one for later...